src/telemetry/telemetry.ts

```diff
@@ -14,7 +14,7 @@ class Telemetry implements ITelemetry {
     this.config = {
       instrumentationKey: this.getInstrumentationKey(),
       disableExceptionTracking: true,
-      disableTelemetry: true,
+      disableTelemetry: false,
```

Comment:
i think this code used to check for instrumentkey before.  you should just revert back to the version before Michael's change